### PR TITLE
Today

### PR DESCRIPTION
**Pull request by cristicrisan07**

### CODE REVIEW

File: Crisan Cristian Alexandru/07 3.8/main.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include <stdio.h>
+#include <stdlib.h>
+/*3.8. Write a function to compute the product of two polynomials,
+ using only pointers and pointer expressions. */
+void PolynomRead(int *Polynom,int PolGr)
+{
+    for(int i=PolGr;i>=0;i--)
+    { printf(" The coefficient of x^%d is:",i);
```
Comment (Alex-Cosma) [inline Crisan Cristian Alexandru/07 3.8/main.c:8]: Ugly formatted code, please properly format

**Review by Alex-Cosma — COMMENTED** [review]
Good work

File: Crisan Cristian Alexandru/07 3.8/main.c (hunk 1 above)
Comment (cristicrisan07) [reply Crisan Cristian Alexandru/07 3.8/main.c:8]: What exactly is ugly? The names of the variables, or the layout of the code?

Comment (Alex-Cosma) [reply Crisan Cristian Alexandru/07 3.8/main.c:8]: The layout of the code! 
Variables seem to be consistently named
Pretty please format code

**Pull request merged**